docs/components/checkbox-group-field/demo/base-demo.md

```diff
@@ -0,0 +1,38 @@
+```hbs template
+<Form::CheckboxGroupField
+  @label='Label'
+  @hint='Extra information about the field'
+  @name='options'
+  @value={{this.groupValue}}
+  @onChange={{this.updateValue}}
+  as |group|
+>
+  <group.CheckboxField @label='Option 1' @option='option-1' />
+  <group.CheckboxField
+    @label='Option 2'
+    @option='option-2'
+    @hint='Extra information about the radio'
+  />
+  <group.CheckboxField @label='Option 3' @option='option-3' />
+  <group.CheckboxField
+    @label='Option 4'
+    @option='option-4'
+    @hint='Extra information about the radio'
+  />
+</Form::CheckboxGroupField>
+```
+
+```js component
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class extends Component {
+  @tracked groupValue = [];
+
+  @action
+  updateValue(value) {
+    this.groupValue = value;
```

Comment:
Docs on this below, but `value` here will be an array of strings that'll align with the `@option` clicked.  Consider the following:

- A user clicks "Option 1"
- `onChange` is called, and the `value` here will be `['option-1']` (note: we use the `@option` provided to tell the user which one was clicked!
- User clicks "Option 3"
- `onChange` is now called, and the `value` here will be `['option-1', 'option-3']`
- User clicks "Option 1" again, unchecking the checkbox
- `onChange` will be `['option-3']`